templates/details/libraries/libraries_layout.html

```diff
@@ -63,20 +65,10 @@ <h3>{{ docstrings["name"] }}</h3>
                 <a href="#" class="is-always-color-link has-icon"><i class="p-icon--download">Download</i><span>Download</span></a>
               </li>
               <li class="p-inline-list__item has-icon">
-                <i class="p-icon--revision">Last revision</i>26 June 2019
+                <i class="p-icon--update">Update</i>26 June 2019
```

Comment:
Stupid question, should it be "Update", or "Updated" or "Last updated" or similar?

Reply:
Not a stupid question. `Last updated` is good.
Fixing it now

Reply:
Fixed